### PR TITLE
Fix url error in API page

### PR DESCRIPTION
**Pull request by saneery**
https://github.com/poanetwork/blockscout/issues/2585

## Motivation

There is a bug with the url on calling of API methods. JS code runs a query just to `https://blockscout.com/api` it should be with a full path like this `https://blockscout.com/eth/mainnet/api`

## Checklist for your PR

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex [hunk 1]
```diff
@@ -41,9 +41,10 @@ defmodule BlockScoutWeb.APIDocsView do
     scheme = url_params[:scheme]
 
     if host != "localhost" do
-      scheme <> "://" <> host <> path
+      "#{scheme}://#{host}#{path}"
     else
-      Endpoint.url()
+      port = Application.get_env(:block_scout_web, BlockScoutWeb.Endpoint)[:http][:port]
+      "#{scheme}://#{host}:#{to_string(port)}#{path}"
```
Comment (ayrat555) [inline apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex]: I think path shouldn't be set locally

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex]: There is a problem with `Endpoint.utl/0` locally, it returns URL with 4001 port. 
As I understand it returns https url by default or http if https doesn't exist. In `dev.exs` we have https, so it returns 4001.
https://github.com/poanetwork/blockscout/blob/26270a583c4979f9bbcdb05f8d4e30d9fe95038f/apps/block_scout_web/config/dev.exs#L29-L33

Comment (ayrat555) [reply apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex]: I'm ok with setting a port. the question was about setting a path

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex]: Could you clarify?

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex]: Oh, do you mean about leaving only that? `"#{scheme}://#{host}:#{to_string(port)}"`

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/views/api_docs_view.ex]: @ayrat555 updated

**Pull request merged**